docs/howto/https/index.adoc

```diff
@@ -0,0 +1,64 @@
+
+[#enabling-https_{context}]
+= Enabling HTTPS
+
+{Thorntail} provides several ways of enabling HTTPS, depending on your use case.
+Here, we describe the most common and most straightforward way.
+
+.Prerequisites
+
+* Keystore in the JKS format. This keystore must include a private key and a corresponding certificate.
++
+--
+If you don't have a keystore, you can create one using `keytool`:
+
+[source,bash,options="nowrap"]
+----
+keytool -genkeypair -keystore my-keystore.jks -storepass my-password -alias my-https-key -keypass my-password -dname CN=localhost -sigalg SHA1withRSA -validity 100 -keyalg RSA -keysize 2048
+----
+
+This keystore is only suitable for testing, as the certificate is self-signed.
+
+To obtain a production-ready certificate, trusted by web browsers, you can use link:https://letsencrypt.org/[Let's Encrypt] or any other certificate authority.
+--
+
+.Procedure
+
+. Include the `management` fraction in the `pom.xml` file in your application:
++
+.pom.xml
+[source,xml]
+----
+<dependencies>
+include::pom.xml[tag=management-fraction,indent=2]
+</dependencies>
+----
+
+. Add the HTTPS properties to your application configuration:
```

Comment:
. Add the HTTPS properties to the following configuration file:
src/main/resources/project-defaults.yml

OR
. Add the HTTPS properties to the project-defaults.yml configuration file available at src/main/resources/.